cloudformation/batch-substack.cfn.json

```diff
@@ -250,29 +249,6 @@
                     "iam:PassRole"
                   ],
                   "Resource": [
-                    {
```

Comment:
I think the following resources are still needed for the batch:SubmitJob permission:
```
{	
    "Fn::Sub": "${JobDefinitionSerial}"	
},
{	
    "Fn::Sub": "${JobDefinitionMNP}"	
}
```

Reply:
yep, updated

Reply:
Also tested by assuming that role and running `awsbsub`, `awsbstat`, `awsbqueues`, `awsbout`, `awsbkill` and `awsbhosts`.

```
$ awsbsub -c batch-policy sleep 60
$ awsbsub -c batch-policy -n 2 sleep 60
$ awsbstat -c batch-policy
jobId                                    jobName    status    startedAt                  stoppedAt    exitCode
---------------------------------------  ---------  --------  -------------------------  -----------  ----------
4ff75939-493a-46c4-b57c-b4f3d93e4253 *2  sleep      RUNNABLE  -                          -            -
c94d9339-0ab4-4342-8947-631ad492091a     sleep      RUNNING   2019-05-13T15:03:42-07:00  -            -
```
```
$ awsbqueues -c batch-policy
jobQueueName              status
------------------------  --------
JobQueue-14804b8147bc8bd  VALID
```
```
$ awsbout 9316a3ad-2ed2-49d3-9467-8e4db436fbd7#1 -c batch-policy
2019-05-13T15:04:51-07:00: Job id: 9316a3ad-2ed2-49d3-9467-8e4db436fbd7#1
2019-05-13T15:04:51-07:00: Initializing the environment...
2019-05-13T15:04:51-07:00: Starting ssh agents...
2019-05-13T15:04:51-07:00: Agent pid 7
2019-05-13T15:04:51-07:00: Identity added: /root/.ssh/id_rsa (/root/.ssh/id_rsa)
2019-05-13T15:04:52-07:00: Mounting /home...
2019-05-13T15:04:52-07:00: Mounting shared file system...
2019-05-13T15:04:52-07:00: Starting the job...
```
```
$ awsbkill 4ff75939-493a-46c4-b57c-b4f3d93e4253 -c batch-policy
Your job cancellation request for job (4ff75939-493a-46c4-b57c-b4f3d93e4253) in status (RUNNABLE) has been submitted.
```
```
$ awsbhosts -c batch-policy
ec2InstanceId        instanceType    privateIpAddress    publicIpAddress      runningJobs
-------------------  --------------  ------------------  -----------------  -------------
i-040065ff85d1e52f2  c4.large        10.0.1.250          -                              0
i-0a4c100c45c6b6336  m4.large        10.0.1.187          -                              1
i-07569c9c16f27a042  c4.large        10.0.1.148          -                              0
```